codecov.yaml

```diff
@@ -12,3 +12,5 @@ ignore:
   - "cli/mocks"
   - "assets"
   - "**/_test.go"
+  - "cli/main.go"
+  - "cli/misc/logger/logger.go"
```

Comment:
Can we add `s3_service` to this list, since we will definitely not want to write tests for it?